nfsd/nfsd.config

```diff
@@ -0,0 +1,9 @@
+CONFIG_NFSD=m
```

Comment:
should we simply build `nfsd` as a module as part of the main kernel build @frezbo ?

Reply:
I think we update the the Kconfig and only build the modules that we ship by default in talos, then these extra can be build as separate modules shipped as extensions, so we don't increase initramfs size

Reply:
ah, okay... makes sense.

Reply:
That's the same approach as I'm taking here. Right?

LMK if there is anything else needed to get this approved and merged in.

Reply:
> That's the same approach as I'm taking here. Right?

meant adding to the actual kernel config under `kernel/build/config-<arch>`, I'll try to experiment and see if that approach works for our use case. Might take a while

Reply:
probably the latter, I'll have to experiment to see what works

Reply:
I couldn't currently figure out where `kernel` is being used to build `installer`. If you can point me, I'll be more than happy to run a few experiments of my own.

Reply:
https://github.com/siderolabs/talos/blob/main/Dockerfile#L77-L99 you can update this with your kernel image and do `make installer PUSH=true`

Reply:
quick question: does this work without rpc service running?

Reply:
@TMaYaD does this require rpc service? Else I can fix this up and get this merged.

Reply:
I worked on this a while back and don't remember how I got this working. I'll check and get back to you.